sdk/nodejs/appservice/zMixins.ts

```diff
@@ -435,12 +435,12 @@ export interface InputOutputsArgs {
     /**
      * Input bindings.
      */
-    inputs?: InputBindingSettings[];
+    inputs?: pulumi.Input<InputBindingSettings[]>;
```

Comment:
I changed this from `InputBindingSettings[]` to instead be an `pulumi.Input` of `InputBindingSettings[]`, since depending on if the array of `InputBindingSettings` instances contains any object that has its `binding` set to an HttpTrigger binding, we either add our own binding or not. Since the `binding` property is an `pulumi.Input<TBindingDefintion>`, I need to `.apply` it in order to get to know if its an HTTP Trigger binding. Thus, I needed to make the whole inputs array an `pulumi.input`.

You can find the relevant code in 'zMixins_http.ts', lines 131 onwards. Do you have any other idea on how to inspect the given bindings without requiring this type to change?